src/Microsoft.AspNet.Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static class AuthenticationServiceCollectionExtensions
     {
         public static IServiceCollection AddAuthentication([NotNull] this IServiceCollection services)
         {
+            services.AddCaching();
```

Comment:
This means no `ÌDistributedCache` imported from the constructor :smile: 

If we can't resolve it from the service provider, can we create our own `LocalCache` and use it for `NonceCache`?